#!/usr/bin/env python
PACKAGE = "saliency_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("compute_grip_angle", bool_t, 0, "Compute grip angle on valid objects", False)

gen.add("debug_font_scale", double_t, 0, "Scale font for debug_image", 1.0, 0, 3.0)

exit(gen.generate(PACKAGE, "SVMColorDiscriminator", "SVMColorDiscriminator"))
